src/character.cpp

```diff
@@ -392,6 +375,38 @@ static const trait_id trait_WEB_WALKER( "WEB_WALKER" );
 static const trait_id trait_WEB_WEAVER( "WEB_WEAVER" );
 
 static const std::string flag_PLOWABLE( "PLOWABLE" );
+static const flag_id json_flag_ALARMCLOCK( "ALARMCLOCK" );
```

Comment:
`character.cpp` already includes `flag.h`, maybe move flag constants there?

Reply:
We've tried static definitions in header files before (ramza specifically did, actually) - it causes a significant increase in build times.